src/app/tasks/services/tasks-statuses.service.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { TasksStatusesService } from './tasks-statuses.service';
+
+
+
+describe('tasksStatusesService', () => {
+  const service: TasksStatusesService = new TasksStatusesService();
+
+
+  test('should create TasksStatusesService', () => {
+    expect(service).toBeTruthy();
+  });
+
+  test('should return the right task status', () => {
+    const mockStatus = service.statusToLabel(10);
+    expect(mockStatus).toBe('Processed');
+  });
```

Comment:
No need to mock it.
Plus, `toBe` is generally used to represent a deep equality, for example when you want to know if an object is really the same as one that one provided to the function.

```suggestion
  test('should return the right task status', () => {
    expect(service.statusToLabel(10)).toEqual('Processed');
  });
```